src/Uno.UI/UI/Xaml/Controls/Popup/Popup.cs

```diff
@@ -147,6 +139,66 @@ public LightDismissOverlayMode LightDismissOverlayMode
 			typeof(Popup),
 			new FrameworkPropertyMetadata(defaultValue: default(LightDismissOverlayMode), propertyChangedCallback: (o, e) => ((Popup)o).ApplyLightDismissOverlayMode()));
 
+		/// <summary>
+		/// Gets or sets the element to use as the popup's placement target.
+		/// </summary>
+		public FrameworkElement PlacementTarget
+		{
+			get => (FrameworkElement)GetValue(PlacementTargetProperty);
+			set => SetValue(PlacementTargetProperty, value);
+		}
+
+		/// <summary>
+		/// Identifies the PlacementTarget dependency property.
+		/// </summary>
+		public static DependencyProperty PlacementTargetProperty { get; } =
+			DependencyProperty.Register(
+				nameof(PlacementTarget),
+				typeof(FrameworkElement),
+				typeof(Popup),
+				new FrameworkPropertyMetadata(default(FrameworkElement)));
+
+		/// <summary>
+		/// Gets or sets the preferred placement to be used for the popup, in relation to its placement target.
+		/// </summary>
+		public PopupPlacementMode DesiredPlacement
+		{
+			get => (PopupPlacementMode)GetValue(DesiredPlacementProperty);
+			set => SetValue(DesiredPlacementProperty, value);
+		}
+
+		/// <summary>
+		/// Identifies the DesiredPlacement dependency property.
+		/// </summary>
+		public static DependencyProperty DesiredPlacementProperty { get; } =
+			DependencyProperty.Register(
+				nameof(DesiredPlacement),
+				typeof(PopupPlacementMode),
+				typeof(Popup),
+				new FrameworkPropertyMetadata(default(PopupPlacementMode)));
```

Comment:
I think it's more readable to be explicit enum value instead of `default(EnumType)`.